doc/en/user/source/community/gsr/installing.rst

```diff
@@ -0,0 +1,8 @@
+Installing the GeoServer JSON-LD extension
+==========================================
+  
+ #. Download the extension from the `nightly GeoServer community module builds <https://build.geoserver.org/geoserver/master/community-latest/>`_.
```

Comment:
Please use the `download_community` directive as described in the mailing list.

```
 #. Download the extension from the nightly GeoServer community module builds :download_community:`<gsr>`.
```

Reply:
fixed